sormas-backend/src/main/java/de/symeda/sormas/backend/externalmessage/labmessage/SampleReportFacadeEjb.java

```diff
@@ -90,7 +90,7 @@ public SampleReport fromDto(@NotNull SampleReportDto source, ExternalMessage lab
 		if (source.getTestReports() != null) {
 			List<TestReport> testReports = new ArrayList<>();
 			for (TestReportDto t : source.getTestReports()) {
-				TestReport testReport = testReportFacade.fromDto(t, target, false);
+				TestReport testReport = testReportFacade.fromDto(t, false);
```

Comment:
This change breaks the ability to save detailed lab messages when importing them from the SORMAS-DEMIS-adapter.